README.md

```diff
@@ -60,14 +62,13 @@ interface spdxLicenseConformance {
   };
 }
 
-function licenseIdConformance(
-  licenseID: string
-): { ok: true, value: spdxLicenseConformance } | { ok: false, value: Error };
+function licenseIdConformance(licenseID: string): { ok: true; value: spdxLicenseConformance } | { ok: false; value: Error };
```

Comment:
For me it seem more readable with the previous version, wdyt ?

Reply:
sorry I had prettier activated on my vscode